queries.yaml

```diff
@@ -335,6 +335,60 @@ post_process:
       start_zoom: 4
       properties: [barrier]
 
+  # coarsen the landuse kinds at mid & low zooms to help improve merging.
+  #
+  # do this _early_; before we assign sort_rank and to avoid inconsistency when
+  # we join (e.g: roads) onto landuse and wouldn't want the road landuse_kind to
+  # be different from the polygon in case that makes it the wrong colour.
+  #
+  # **NOTE** WE'RE CHANGING THE KINDS HERE!
+  - fn: vectordatasource.transform.remap
+    params:
+      layer: landuse
+      start_zoom: 0
+      end_zoom: 13
+      property: kind
+      # note that we don't need to remap things to themselves, e.g:
+      # kind=desert stays as-is.
+      remap:
+        # barren
+        quarry: barren
+        scree: barren
+        shingle: barren
+
+        # desert
+        bare_rock: desert
+        sand: desert
+
+        # farmland
+        allotments: farmland
```

Comment:
While I think technically correct as-is, alternatively `village_green` and `allotments` could both be remapped to `urban_area` if that promotes more merging? Would require data stats...

Reply:
Sure, we can map it however you want! The [original request](https://github.com/tilezen/vector-datasource/issues/1779#issuecomment-452452746) was for:

> - **grassland** = 70 (use OSM's `grassland` instead of Marble Cutter's `herbaceous`)
>    - ...
>    - other landuse=[grass,village_green]
> - **farmland**  = 80  (use OSM's `farmland` instead of Marble Cutter's `cultivated`)
>    - ...
>    - other landuse=[...,allotments,...]

Allotments are usually close to (sub)urban areas, so I can see that they would probably merge with nearby `residential` areas. `village_green`, I'm not so sure about - it'll certainly be near houses, but whether there's enough houses for someone to have bothered drawing a `residential` area... I'm not sure.

Reply:
Changed in https://github.com/tilezen/vector-datasource/pull/1790/commits/6c711e2a9805361692c50098ceaf25a6175af12d.

(This one is more to taste, thanks for the change!)